packages/cli/src/commands/gsn-deploy.ts

```diff
@@ -21,10 +21,11 @@ gsnCommander(['n', 'f', 'm', 'g', 'l'])
   .option('--relayRegistrar <address>', 'relayRegistrar')
   .option('--environmentName <string>', `name of one of the GSN supported environments: (${Object.keys(EnvironmentsKeys).toString()}; default: ethereumMainnet)`, EnvironmentsKeys.ethereumMainnet)
   .option('--burnAddress <string>', 'address to transfer burned stake tokens into', constants.BURN_ADDRESS)
+  .option('--stakingToken <string>', 'default staking token to use. (wrapped eth)')
+  .option('--minimumTokenStake <number>', 'minimum staking eth value (in wrapped eth staking token)', '1')
```

Comment:
This can be denominated in any token passed to `stakingToken`, why do you write `wrapped eth` explicitly?

Reply:
this is denominated in the stakingToken units (which are supposed to be WETH..)
so it is just "minimum staking value"